src/wp-includes/rest-api.php

```diff
@@ -1539,6 +1539,67 @@ function rest_stabilize_value( $value ) {
 	return $value;
 }
 
+/**
+ * Gets the possible error of combining operation.
+ *
+ * @since 5.6.0
+ *
+ * @param array  $value  The value to validate.
+ * @param string $param  The parameter name, used in error messages.
+ * @param array  $errors The errors array, to search for possible error.
+ * @return WP_Error      The combining operation error.
+ */
+function rest_get_combining_operation_error( $value, $param, $errors ) {
+	// If there is only one error, simply return it.
+	if ( 1 === count( $errors ) ) {
+		$reason = $errors[0]['error_object']->get_error_message();
+
+		/* translators: 1: Parameter, 2: Reason. */
+		return new WP_Error( 'rest_invalid_param', sprintf( __( '%1$s does not match any schema. Reason: %2$s' ), $param, $reason ) );
+	}
+
+	// Filter out all errors related to type validation.
+	$filtered_errors = array();
+	foreach ( $errors as $error ) {
+		$best_type = rest_get_best_type_for_value( $value, array( $error['schema']['type'] ) );
+		if ( $best_type === $error['schema']['type'] ) {
+			$filtered_errors[] = $error;
+		}
+	}
+
+	// If there is only one error left, simply return it.
+	if ( 1 === count( $filtered_errors ) ) {
+		$reason = $filtered_errors[0]['error_object']->get_error_message();
+
+		/* translators: 1: Parameter, 2: Reason. */
+		return new WP_Error( 'rest_invalid_param', sprintf( __( '%1$s does not match any schema. Reason: %2$s' ), $param, $reason ) );
+	}
+
+	// If there are only errors related to object validation, try choosing the most appropriate one.
+	if ( count( $filtered_errors ) > 1 && 'object' === $filtered_errors[0]['schema']['type'] ) {
+		$result = null;
+		$number = 0;
+
+		foreach ( $filtered_errors as $error ) {
+			if ( isset( $error['schema']['properties'] ) ) {
+				$n = count( array_intersect_key( $error['schema']['properties'], $value ) );
+				if ( $n > $number ) {
+					$result = $error['error_object'];
+					$number = $n;
+				}
+			}
+		}
+
+		$possible_reason = $result->get_error_message();
+
+		/* translators: 1: Parameter, 2: Possible reason. */
+		return new WP_Error( 'rest_invalid_param', sprintf( __( '%1$s does not match any schema. Possible reason: %2$s' ), $param, $possible_reason ) );
+	}
+
+	/* translators: 1: Parameter. */
+	return new WP_Error( 'rest_invalid_param', sprintf( __( '%1$s does not match any schema.' ), $param ) );
```

Comment:
```suggestion
	return new WP_Error( 'rest_invalid_param', sprintf( __( '%1$s does not match any of the expected formats.' ), $param ) );
```

Reply:
If we have a valid `title` for each schema, let's include those titles in the error message instead. So it would read something like `param is not a valid Crop, Rotation and Zoom`. That can be done using `wp_sprintf` and `%l`.